event_generation/setup.py

```diff
@@ -0,0 +1,19 @@
+from setuptools import setup, find_packages
+
+setup(
+    name="event_generation",
+    author="Alberto Jaen",
+    description="Locust code to send events to AWS Kinesis",
+    packages=find_packages(),
+    python_requires=">=3.9",
```

Comment:
Apache Flink 1.15.2 is compatible up to python 3.8. Maybe we should change the python requirement to avoid nuisances with the local environment